bench/Suites.hs

```diff
@@ -0,0 +1,30 @@
+module Suites where
+
+import Base
+import Juvix.Prelude
+import Variants
+
+suites :: [Suite]
+suites =
+  map
+    defaultSuite
+    [ "mergesort",
+      "fibonacci",
+      "maybe"
+    ]
+    <> [ Suite "fold" (allVariantsExcept [C] [CoreEval]),
+         Suite "mapfold" (allVariantsExcept [C] [CoreEval])
+       ]
+
+-- "mapfun" -- juvix crashes: Address boundary error
+-- "ackermann", -- juvix crashes
+-- "combinations", -- juvix crashes:  call stack exhausted
+-- "cps", juvix: call stack exhausted
+-- "prime" -- juvix: Address boundary error
+
```

Comment:
Why aren't those benchmarks included? That the current implementation of juvix crashes is a problem with the old backend, but the benchmarks themselves test reasonable things. The crash should just be noted on the plot / in the results.

Reply:
I have included them now. The native and wasm Juvix variants have been excluded since it is not possible to benchmark functions that may crash using the criterion library.